policies/ip-policy.md

```diff
@@ -50,7 +50,7 @@ Project leaders should ensure compliance with this policy and provide clear guid
 
 ### Copyright Statements
 
-Using [copyright statements in your code][code-copyright] is a practice with mixed adoption.
+Using is a practice with mixed adoption.
```

Comment:
This doesn't make sense to me. Should the link have been removed but the text remained?